_layouts/product-get-started-old.html.slim

```diff
@@ -46,9 +46,14 @@ br
           = page._extra_installation_instructions
     - else 
       li
-        asciidoc:
-          #{page.product.abbreviated_name} is only available through a full Red Hat subscription. Download #{page.product.abbreviated_name} by
-          https://access.redhat.com/jbossnetwork/restricted/listSoftware.html[logging in to the Red Hat Customer Portal]  and downloading it from there.
+        - if page.product.id == 'brms' || page.product.id == 'bpmsuite'
```

Comment:
OK, figured out the problem. See the line with 'if page.product.default_download_artifact'. This displays the offending text if that key is null. Now that we use a REST API to get 'default_download_artifact' it will always be null at page build time. I guess the new get-started pages are not susceptible to this. I think we need to figure out a better way, at build-time to decide wether to show this or not.